src/main/resources/templates/AdvisorRHEL/resolvedRecommendationInstantEmailBody.html

```diff
@@ -1,5 +1,5 @@
-{#include Advisor/insightsEmailBody}
-{#content-title}Advisor instant notification - {action.timestamp.toUtcFormat()}{/content-title}
+{#include AdvisorRHEL/insightsEmailBody}
+{#content-title}RHEL - Advisor Instant Notification - {action.timestamp.format('d MMM uuuu')}{/content-title}
```

Comment:
We would need UXD approval if this change, and then applied everywhere, we need UTC time

Reply:
in the last commit I took the old text from the plain advisor.